src/scrapers/base-scraper-with-browser.ts

```diff
@@ -75,6 +75,14 @@ function createGeneralError(): ScraperScrapingResult {
   };
 }
 
+async function safeCleanup(cleanup: () => Promise<void>) {
+  try {
+    await cleanup();
+  } catch (e) {
+    debug(`Cleanup function failed: ${(e as Error).message}`);
+  }
```

Comment:
```suggestion
  } catch (e) {
    debug(`Cleanup function failed`, e);
  }
```

https://github.com/eshaham/israeli-bank-scrapers/pull/968#discussion_r2250087298